cloudbuild/nightly/scripts/python-scripts/delete_buckets_and_clusters.py

```diff
@@ -0,0 +1,158 @@
+# Copyright 2023 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#            http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+
+import argparse
+from absl import logging
+from absl import app
+from collections.abc import Sequence
+import os
+from google.cloud import dataproc_v1
+
+
+def clear_all_dataproc_dependencies(project_id, cluster_name, region):
+    delete_active_jobs(project_id, cluster_name, region)
+    delete_cluster_buckets(project_id, cluster_name, region)
+    delete_cluster(project_id, cluster_name, region)
+
+
+def delete_active_jobs(project_id, cluster_name, region):
+    """Function to list and cancel all running jobs on dataproc cluster.
```

Comment:
Lets keep consistent names. The job is being canceled, not deleted.
Also, and more importantly, why do you need to explicitly cancel the job? Didnt you add an async timer on the Flink application itself to end at some point?

Reply:
I see https://github.com/GoogleCloudDataproc/flink-bigquery-connector/blob/main/flink-connector-bigquery-integration-test/src/main/java/com/google/cloud/flink/bigquery/integration/BigQueryIntegrationTest.java#L225

Reply:
yes. Makes Sense. Fixed.